lib/jxl/jxl_test.cc

```diff
@@ -1699,5 +1699,26 @@ TEST(JxlTest, JXL_X86_64_TEST(StreamingSamePixelsAlpha)) {
   EXPECT_TRUE(jxl::test::SamePixels(ppf_out, ppf_out_streaming));
 }
 
+TEST(JxlTest, JXL_X86_64_TEST(StreamingSamePixelsLossless)) {
```

Comment:
maybe it would be worth converting it to a parameterized test now and adding a few more modular settings like palette and modular color space and such

Reply:
Good point - will do that as a follow-up PR (after the group sizes also go in).

Ideally it should also include the lossy case and just test various different combinations...